po/tr.po

```diff
@@ -3882,18 +3955,18 @@ msgid ""
 "covered later."
 msgstr ""
```

Comment:
A number of strings are not translated here. I don't know if 100% coverage is required or what, but the PR says "fully" and there are a good chunk of empty strings near here.

Reply:
Okay got it. I wasn't following where all the pieces were coming from or what the expectations on completeness were, just spot checking.

Kolay gelsin.

Reply:
Teşekkür ederim :)

Reply:
Empty strings are okay in a PO file — that just means that the entry isn't translated yet. The English text will be used in the interim.